actfw_core/__init__.py

```diff
@@ -2,7 +2,7 @@
 from pathlib import Path
 from typing import Any, Callable, Dict, List
 
-from . import capture, linux, task, unicam_isp_capture  # noqa: F401
+from . import capture, linux, system, task, unicam_isp_capture  # noqa: F401
```

Comment:
conflict with https://github.com/Idein/actfw-core/pull/81.

@KeenS @eguchi1904 Which is it better to close #81 or delete this commit?

Reply:
Sorry, I've already merged #81 . I guess `git merge` so clever that they won't conflict.